Backend/NoCode.FlowerShop.Application/Flowers/Create/CreateFlowerCommandHandler.cs

```diff
@@ -0,0 +1,37 @@
+using ErrorOr;
+using MediatR;
+using NoCode.FlowerShop.Application.Common.Interfaces.Persistence;
+using NoCode.FlowerShop.Domain;
+using NoCode.FlowerShop.Domain.Common.ErrorsCollection;
+
+namespace NoCode.FlowerShop.Application.Flowers.Create;
+
+public sealed class CreateFlowerCommandHandler 
+    : IRequestHandler<CreateFlowerCommand, ErrorOr<CreateFlowerResult>>
+{
+    private readonly IFlowerRepository _flowerRepository;
+
+    public CreateFlowerCommandHandler(IFlowerRepository flowerRepository)
+    {
+        _flowerRepository = flowerRepository;
+    }
+
+    public async Task<ErrorOr<CreateFlowerResult>> Handle(CreateFlowerCommand request, CancellationToken cancellationToken)
+    {
+        var isUniqueName = await IsUniqueName(request.Name);
+        if (!isUniqueName)
+            return Errors.Flowers.DuplicateName;
+
+        var flowerToCreate = new Flower(request.Name, request.ImageUrl);
+
+        var createdFlower = await _flowerRepository.AddAsync(flowerToCreate, CancellationToken.None);
```

Comment:
```csharp
        var createdFlower = await _flowerRepository.AddAsync(flowerToCreate);
```

Reply:
IDE suggested to explicitly tell that CacncellationToken is not used here, but to match the overall coding style, will ignore the suggestion from IDE and fix it